Misc/NEWS.d/3.13.0b1.rst

```diff
@@ -1346,9 +1346,10 @@ urllib.
 .. nonce: du4UKW
 .. section: Library
 
-Callbacks registered in the :mod:`tkinter` module now take arguments as
-various Python objects (``int``, ``float``, ``bytes``, ``tuple``), not just
-``str``. To restore the previous behavior set :mod:`!tkinter` module global
+Add support for passing arguments to callbacks registered in the :mod:`tkinter`
+module as various Python objects (``int``, ``float``, ``bytes``, ``tuple``),
+corresponding to the original Tcl value, not just ``str``, and enable this by default.
+To restore the previous behavior set :mod:`!tkinter` module global
```

Comment:
This does not make sense when the value already is 1.

Reply:
It is an edited NEWS entry for 3.13.0b1.

Reply:
The change to 3.13 what's new suggests that the change to wantobjects will be backported also.  'Restoring' 1 to 1 is what I questioned.

Reply:
This is an edited NEWS entry for 3.13.0b1. We cannot simply remove it, because it is the part of the history, and 3.13.0b1 was already released with the bug.

But I reworded it, focusing on the _possibility_ to change the behavior and adding that it was changed by default (not for long, for 3.13.0b1 only). So "restoring" here refers to the changed default. How to write this better? Maybe just remove all starting from "To restore"? And add the description of how to enable the experimental feature in the new NEWS entry for 3.13.0b2?